common/lib/xmodule/xmodule/modulestore/tests/test_xml.py

```diff
@@ -35,7 +35,8 @@ def test_xml_modulestore_type(self):
         store = XMLModuleStore(DATA_DIR, source_dirs=[])
         self.assertEqual(store.get_modulestore_type(), ModuleStoreEnum.Type.xml)
 
-    def test_unicode_chars_in_xml_content(self):
+    @patch('xmodule.tabs.CourseTabList.initialize_default')
+    def test_unicode_chars_in_xml_content(self, _initialize_default_mock):
```

Comment:
If you aren't going to do anything with the mock, you can just pass `Mock()` as the second argument to `@patch`, so that you don't have to have an argument to the method.